PWGHF/TableProducer/candidateSelectorD0.cxx

```diff
@@ -134,6 +135,12 @@ struct HfCandidateSelectorD0 {
     // candidate DCA
     // if (candidate.chi2PCA() > cuts[pTBin][1]) return false;
 
+    // candidate chi2
+    // if constexpr (ReconstructionType == useKFParticle) {
+    //   if (candidate.kfTopolChi2OverNdf() > cuts->get(pTBin, "topological chi2overndf as D0")) return false;
+    //     return false;
+    // }
```

Comment:
Please add a comment specifying the condition for uncommenting this block, i.e. when it can be used.